olga/analytics/create_test_fixture.py

```diff
@@ -0,0 +1,135 @@
+"""
+Utilities, that create test data for charts and map.
+Needed to be deleted in production as well as `olga/analytics/test_fixture.json`.
```

Comment:
> Why we need to delete it in production? Do you mean, delete them from database after migraion? Why? They populate database automatically?

`create_test_fixture.py` has a lot of pylint issues. It needs to be rewritten. 
Additional bad is a courses data it is equal to students data. It produces graph above.

![selection_072](https://user-images.githubusercontent.com/22666467/28414703-911189dc-6d55-11e7-9192-4f499e1d8f03.png)

Look, you cannot see one of them.